config/config.go

```diff
@@ -1293,6 +1294,23 @@ func (c *Config) CheckLoggerConfig() error {
 	return nil
 }
 
+func (c *Config) checkDatabaseConfig() error {
```

Comment:
Please add a check here to see if the user has specified an unsupported driver and return an error accordingly